016-Power_digit_sum.js

```diff
@@ -0,0 +1,39 @@
+// 2^15 = 32768 and the sum of its digits is 3 + 2 + 7 + 6 + 8 = 26.
+// What is the sum of the digits of the number 2^1000?
+
+// In JavaScript, integer can't use more than 53 bits.
+
+function sumOfPowerDigits(exponent) {
+  let sum = 0;
+  let res = [ 1 ];
+
+  for (let i = 0; i < exponent; ++i) {
+    res = multiple2(res);
+  }
+
+  for (let i = 0; i < res.length; ++i) {
+    sum += res[i];
+  }
+
+  return sum;
+}
+
+function multiple2(num) { // num is an array
+  const len = num.length;
+  let res = new Array(len);
```

Comment:
This creates the a new array on each multiplication.
It can also be done inplace, but up to you to decide if you wanna do that.

Reply:
I wanna do that :)